### PR TITLE
Replace cbor-custom with airnode-abi

### PR DESCRIPTION
**Pull request by andreogle**
### Changes

1. See title. Replaces `cbor-custom` with `@airnode/airnode-abi` in the node

### CODE REVIEW

File: packages/node/src/core/evm/abi-encoding.test.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    );
+    expect(res).toEqual({
+      from: 'ETH',
+      amount: '1000',
+    });
+  });
+
+  it('returns null if parameters are empty', () => {
+    const res = safeDecode('0x');
+    expect(res).toEqual(null);
```
Comment (bbenligiray) [inline packages/node/src/core/evm/abi-encoding.test.ts:16]: Wouldn't `{}` be more accurate here?

File: packages/node/src/core/evm/abi-encoding.test.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  });
+
+  it('returns an empty object if parameters are falsey', () => {
+    const res = safeDecode('');
+    expect(res).toEqual({});
```
Comment (bbenligiray) [inline packages/node/src/core/evm/abi-encoding.test.ts:21]: and `null` here because `"0x"` is a valid empty encoding and `""` is not

Comment (bbenligiray) [reply packages/node/src/core/evm/abi-encoding.test.ts:21]: For context, if you declare a `bytes` variable in a contract and retrieve it before initializing it, it returns `"0x"`

**Review by bbenligiray — COMMENTED** [review]
Some comments about how empty parameters are treated

File: packages/node/src/core/evm/abi-encoding.test.ts (hunk 1 above)
Comment (andreogle) [reply packages/node/src/core/evm/abi-encoding.test.ts]: Yeah good point. I added some special cases for zero addresses to the airnode-abi package here: https://github.com/api3dao/airnode/pull/159/commits/db4adfe1257d39109d5f7ed0515474e17a54438b

I don't know if I need to cover both `AddressZero` and `HashZero` but it feels a bit safer

Comment (bbenligiray) [reply packages/node/src/core/evm/abi-encoding.test.ts]: I'm not sure about that. `AddressZero` and `HashZero` are not any different than `"this is not a valid ABI encoding"`.

Comment (andreogle) [reply packages/node/src/core/evm/abi-encoding.test.ts]: Ok so just handling `0x` is all we need then?

Comment (bbenligiray) [reply packages/node/src/core/evm/abi-encoding.test.ts]: I think so

Comment (andreogle) [reply packages/node/src/core/evm/abi-encoding.test.ts:27]: Decoding `AddressZero` or `HashZero` now throws an error

**Review by bbenligiray — APPROVED** [review]

**Pull request merged**